transfer/transfer-06-consumer-pull-http/README.md

```diff
@@ -274,10 +272,10 @@ Sample output:
   "@id": "31f6d748-d35b-4dec-9e34-d141fd17b458",
   "@type": "dcat:Catalog",
   "dcat:dataset": {
-    "@id": "f31a04bf-3575-4f60-9795-9550e979143a",
+    "@id": "assetId",
     "@type": "dcat:Dataset",
     "odrl:hasPolicy": {
-      "@id": "1:assetId:902a192a-dcc6-49d0-8c9f-17daa9303730",
+      "@id": "MQ==:YXNzZXRJZA==:YTc4OGEwYjMtODRlZi00NWYwLTgwOWQtMGZjZTMwMGM3Y2Ey",
```

Comment:
Shouldn't this contain the contract definition and asset ids? same for sample 7

Reply:
it does, but it is encoded now, see https://github.com/eclipse-edc/Connector/issues/3211

Reply:
ah alright, I missed that